### PR TITLE
movie: Red Planet (2000)

### PR DESCRIPTION
**Pull request by paul-sheridan**
Better than Red Dwarf. Rating 2/5.

### CODE REVIEW

File: notes/stories/film/film-scifi-2000s.st.txt [hunk 1]
```diff
@@ -497,6 +497,51 @@ cryptid [A scientist explained that the strange insects were actually prehistori
...
+human vs. technology [The protagonists were stalked by the advanced, but non-sentient military robot AMEE as they trekked across the surface of Mars.]
+
+:: Minor Themes
+overpopulation [It was stated in the opening narration that the Earth had become overpopulated by the year 2000.]
+pollution [It was stated in the opening narration that humanity had "polluted and poisoned" the Earth beyond the point where it could be cleaned up.]
+space station [The MARS-1 crew departed on their mission to the Red Planet from a large space station situated in high Earth orbit.]
+interplanetary space travel [The first part of the film concerned followed Commander Kate Bowman and her crew as they went on the first manned mission to Mars.]
+speculative spaceship [The protagonists flew to Mars in an experimental, fusion powered spaceship.]
+speculative robot [The anthropomorphic, autonomous military robot AMEE went haywire and started stalking the protagonists.]
```
Comment (odinlake) [inline notes/stories/film/film-scifi-2000s.st.txt:535]: https://www.themeontology.org/theme.php?name=autonomous%20weapons%20system ?

**Review by odinlake — APPROVED** [review]
Left one suggestion but merge at will.

File: notes/stories/film/film-scifi-2000s.st.txt (hunk 1 above)
Comment (paul-sheridan) [reply notes/stories/film/film-scifi-2000s.st.txt:535]: I added it. But I suppose we should consider at some point whether this should apply for individual robots or just for larger systems, like that one from TNG The Arsenal of Freedom.

**Comment by odinlake** [comment]
It is, of course, themer’s choice whether the situation in this story
represented the contemporary social issue of humanity creating autonomous
killing devices. The description sounded (to me) like it was. Obviously we
don’t use the theme in every space opera with killer robots because we
don’t think anyone is likely to consider the contemporary social issue
while watching those stories.

**Pull request merged**